repowiz-infrastructure/src/main/groovy/life/qbic/repowiz/RepositoryCreator.groovy

```diff
@@ -0,0 +1,15 @@
+package life.qbic.repowiz
+
+class RepositoryCreator {
+
+    Repository createRepository(Map data){
```

Comment:
How does you exception handling look like here? Like in case properties in the map are missing?

Reply:
I added exception handling with b2afb82, the repository is validated when the json file is loaded with the json parser through the schema thus at this point of the code the repository must! be valid since all fields are required